workflow-v2.1.0-e2e/README.md

```diff
@@ -0,0 +1,12 @@
+# Workflow 2.0.0 e2e Tests (parallel)
+
+End-to-end tests for the Deis Workflow open source PaaS, executed in parallel.
+
+NOTE: This assumes a fresh `helmc install` of Deis v2. The test suite creates
+an initial user with admin privileges (who is deleted when tests complete).
+See https://github.com/deis/workflow-e2e for more detail.
+
+```console
+$ helmc uninstall -n deis -y workflow-e2e ; helmc install workflow-e2e
+$ kubectl --namespace=deis logs -f workflow-e2e tests
```

Comment:
This line needs to be updated to reference the actual workflow-v2.1.0-e2e container, as we did for [the v2.0.0 e2e chart](https://github.com/deis/charts/tree/master/workflow-v2.0.0-e2e). Otherwise it won't run:
```console
$ kubectl --namespace=deis logs -f workflow-e2e tests
Error from server: pods "workflow-e2e" not found